### PR TITLE
Jason Kelce (combined)

### PR DESCRIPTION
**Pull request by jjasghar**
This is the combined  Jason Kelce PRs from Red Hat Summit 2024.

This superseeds #1028 #946

### CODE REVIEW

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  question: What's Jason Kelce's brother's name?
+- answer: The total number of games Kelce has played is 193
+  question: how many games Jason Kelce played so far?
+- answer: Center
+  question: What position does Jason Kelce play?
+- answer: Kelce played 16 games in 2015
+  question: How many games Jason Kelce played in the year 2015?
+- answer: November 5, 1987
+  question: when did Jason Kelce was born?
+task_description: ' '
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:23]: This need a proper task description.

For example:

```suggestion
task_description: 'To teach a language model about the person Jason Kelce'
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+created_by: jjasghar
+version: 2
+domain: athlete
+seed_examples:
+- answer: Clevelan Heights, Ohio
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:5]:
```suggestion
- answer: Cleveland Heights, Ohio
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  question: Where was Jason Kelce born?
+- answer: Philadelphia Eagles
+  question: Which NFL team drafted Jason Kelce?
+- answer: None
+  question: What other teams did Jason Kelce play for besides Philadelphia?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:10]:
```suggestion
  question: What other NFL teams did Jason Kelce play for besides Philadelphia?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- answer: The sixth round
+  question: What round was Jason Kelce drafted?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:12]:
```suggestion
  question: What round of the NFL draft was Jason Kelce drafted?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+- answer: Travis Kelce
+  question: What's Jason Kelce's brother's name?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:14]:
```suggestion
  question: What is the name of Jason Kelce's brother?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- answer: The total number of games Kelce has played is 193
+  question: how many games Jason Kelce played so far?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:16]:
```suggestion
  question: How many NFL games Jason Kelce played?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+- answer: Center
+  question: What position does Jason Kelce play?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:18]:
```suggestion
  question: What NFL position does Jason Kelce play?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+- answer: Kelce played 16 games in 2015
+  question: How many games Jason Kelce played in the year 2015?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:20]:
```suggestion
  question: How many NFL games did Jason Kelce play in the year 2015?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+- answer: November 5, 1987
+  question: when did Jason Kelce was born?
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:22]:
```suggestion
  question: When was Jason Kelce born?
```

File: knowledge/sports/athletes/jason_kelce/qna.yaml [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+task_description: ' '
+document:
+ repo: https://github.com/juliadenham/Summit_knowledge
+ commit: 91e2f2d887a256c391ff36933f422c064524968c
+ patterns:
+    - jason_kelce.md
```
Comment (bjhargrave) [inline knowledge/sports/athletes/jason_kelce/qna.yaml:28]:
```suggestion
document:
  repo: https://github.com/juliadenham/Summit_knowledge
  commit: 91e2f2d887a256c391ff36933f422c064524968c
  patterns:
    - jason_kelce.md
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Comment by jjasghar** [comment]
I dare say that with your edits, @bjhargrave, you may have just created the "best" version to show as an example `qna.yaml` file for people to learn how to ask the questions.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**